introduction.md

```diff
@@ -1,2 +1,16 @@
 My name is Justin.  My favourite code Code School path is Elixir.  I have
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?